libs/wingc/src/type_check.rs

```diff
@@ -3317,11 +3317,19 @@ impl<'a> TypeChecker<'a> {
 				self.update_known_inferences(&mut return_type, &scope.span);
 			}
 
-			let mut has_return_stmt_visitor = HasReturnStatementVisitor::default();
```

Comment:
I think the code for `HasReturnStatementVisitor` can be removed

Reply:
Sure! I'll do, I didn't want to mess with old code but if I have this confirmation I'll proceed